components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -19,16 +19,14 @@
 #import "MaterialTypography.h"
 #import "supplemental/MDCBannerView.h"
 
-static const CGFloat exampleListTableViewHeight = 300.0f;
-static const CGFloat exampleBannerContentMultiLineWidth = 320.0f;
-static const CGFloat exampleBannerContentSingleLineWidth = 350.0f;
+static const CGFloat exampleListTableViewHeight = 160.0f;
+static const CGFloat exampleBannerContentPadding = 10.0f;
 static NSString *const exampleShortText = @"tristique senectus et";
 static NSString *const exampleLongText =
-    @"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.";
```

Comment:
What does this string change have to do with the latest API changes?

Reply:
Updated the description to reflect the constant change here.